bin/dry-run.rb

```diff
@@ -474,10 +474,7 @@ def log_conflicting_dependencies(conflicting_dependencies)
   commit: $options[:commit]
 )
 
-always_clone = Dependabot::Utils
-               .always_clone_for_package_manager?($package_manager)
```

Comment:
wondering if we would ever need an 'opt-out' option (the inverse of our current state). On 2nd thoughts; I'll default to YAGNI. And as @jakecoffman mentioned, we do have Actions as a solution.